config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
   namespace :api do
     namespace :v1 do
+      # namespace :users do
+      #   post '/', to: 'users#create'
+      # end
+      post '/users', to: 'users#create'
```

Comment:
not sure why this namespace was not working, will do further refactoring when project is complete